lib/utils/getSelectorMeaningfulSource.mjs

```diff
@@ -0,0 +1,45 @@
+import selectorParser from 'postcss-selector-parser';
+
+/**
+ * Get the source index of a selector, excluding leading whitespace.
```

Comment:
question

I thought it was supposed to remove comments too?

Reply:
It does, comment was incorrect, should be clearer now. 🙇